data-processing/scripts/convert_scandi_wiki_to_jsonlgz.py

```diff
@@ -0,0 +1,62 @@
+"""
+Convert jsonl files in tarfile from this format:
+
+{
+    "id": "...",
+    "url": "...",
+    "title": "...",
+    "text": "...",
+}
+
+To this format:
+{
+    "id": "...",             # MANDATORY: source-specific identifier
+    "text": "foo",           # MANDATORY: textual content of the document
+    "source": "...",         # MANDATORY: source of the data, such as peS2o, common-crawl, etc.
+    "added": "...",          # OPTIONAL: timestamp we acquired this data (time file was created), specified as YYYY-MM-DDTHH:MM:SS.TIMEZONE e.g 2021-04-13T12:52:46.000Z
+    "created": "..."         # OPTIONAL: timestamp when orig document was created (best-guess if not available), should be specified as a range; "YYYY-MM-DDTHH:MM:SS.TIMEZONE, YYYY-MM-DDTHH:MM:SS.TIMEZONE"
+    "metadata": {            # OPTIONAL: source-specific metadata
+        "sub-source": "...", # OPTIONAL: E.g. "newspaper_ocr"
+        ...
+    }
+}
+
+"""
+
+import datetime
+import gzip
+import json
+from pathlib import Path
+import tarfile
+
+def main():
+    input_dir = Path("/directory-with-scandi-wiki.tar.gz-inside/")
+    output_dir = Path("scandi-wiki")
+    time_added = datetime.datetime.now(datetime.UTC).strftime("%Y-%m-%dT%H:%M:%S.000Z")
+    wiki_created = "2001-01-15T00:00:00.000Z"
```

Comment:
Isn't this quite old wiki content?

Reply:
The raw dataset does not have timestamps and this date was the one I found for when the first version of Wikipedia was created.